### PR TITLE
PT Capacity Update

### PR DESCRIPTION
**Pull request by Musketeiro**
PT installed capacity update based on official source.

### CODE REVIEW

**Comment by Musketeiro** [comment]
In case there is any doubt about the hydro values:

8163 = Total Hydro.
3585 = Hydro Storage (Pumping/bombagem, as it is called on the REN website)
4578 = Hydro Power (8163-3585).

File: config/zones.json [hunk 1]
```diff
@@ -3922,18 +3922,18 @@
...
+      "hydro": 4578,
+      "hydro storage": 3585,
       "nuclear": 0,
       "oil": 0,
-      "solar": 1364,
-      "wind": 5528
+      "solar": 1616,
+      "wind": 5389
     },
-    "contributors": ["corradio", "nessie2013", "Falconet8"],
```
Comment (VIKTORVAV99) [inline config/zones.json:3936]: Why did you remove Falconet8 as a contributor?

File: DATA_SOURCES.md [hunk 2]
```diff
@@ -327,7 +327,7 @@ For many European countries, data is available from [ENTSO-E](https://transparen
 - Peru: [IRENA](https://www.irena.org/publications/2022/Apr/Renewable-Capacity-Statistics-2022)
 - Philippines: [Philippine Department of Energy](https://www.doe.gov.ph/electric-power/list-existing-power-plants-june-30-2022)
 - Poland: [ARE](https://www.are.waw.pl/badania-statystyczne/wynikowe-informacje-statystyczne/publikacje-miesieczne#informacja-statystyczna-o-energii-elektrycznej)
-- Portugal
+- Portugal: [REN](https://datahub.ren.pt/pt/eletricidade/balanco-mensal/)
   - Coal: [TEJO](https://www.tejoenergia.com/en/aboutus/)
   - Other: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
```
Comment (VIKTORVAV99) [inline DATA_SOURCES.md:332]:
```suggestion
- Portugal: 
  - Biomass, Hydro, Solar, Wind: [REN](https://datahub.ren.pt/pt/eletricidade/balanco-mensal/)
  - Other: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
```

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
I just have a suggestion for the source formatting and a question about the contributor change that needs to be addressed but otherwise the changes looks good.

File: config/zones.json (hunk 1 above)
Comment (Musketeiro) [reply config/zones.json:3936]: Falconet8 is me. I changed the username.
What did you mean about the source?

Comment (Musketeiro) [reply config/zones.json:3936]: I see what you meant about the source. I pressed commit.

Comment (Musketeiro) [reply config/zones.json:3936]: Looks like something failed regarding the review checks. I don't understand any of it so I'll leave to someone else.

**Review by VIKTORVAV99 — APPROVED** [review]

**Comment by Musketeiro** [comment]
Thanks, @VIKTORVAV99.
